spec/stringProcessing/createRegexFromArray.js

```diff
@@ -1,7 +1,50 @@
+/* eslint-disable no-irregular-whitespace */
+/* eslint-disable no-useless-escape */
 import arrayToRegex from "../../src/stringProcessing/createRegexFromArray.js";
 
 describe( "a test creating a regex from an array with strings", function() {
+	// This test is old and it should not pass, but it does!
```

Comment:
Nice catch!
As discussed, the `toMatch` executes the regex instead of comparing it. Please remove the whole test.